examples/scene_create/Cargo.toml

```diff
@@ -10,4 +10,4 @@ crate-type = ["cdylib"]
 
 [dependencies]
 gdnative = { path = "../../gdnative" }
-euclid = "0.22.1"
+glam = "*"
```

Comment:
This is rather unusual -- I would use the latest version, so upgrading breaking versions (minor for 0.*) is manual.